data/completion/etelpmoc.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
```

Comment:
Curious, why is the file called this way? And a small comment in the header what its actually doing would be great.